Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Settings/GetIshSetting.Tests.ps1

```diff
@@ -94,10 +96,12 @@ Describe “Get-IshSetting" -Tags "Read" {
 			$ishFields = Get-IshSetting -IshSession $ishSession -RequestedMetadata $requestedMetadata
 			$ishSession.DefaultRequestedMetadata = $oldDefaultRequestedMetadata
 			$ishFields.GetType().Name | Should BeExactly "Object[]"
-			if ((Get-IshVersion).MajorVersion -le 13) { $ishFields.Length | Should Be 31 }
-			if ((Get-IshVersion).MajorVersion -ge 14) { $ishFields.Length | Should Be 35 } 
-				# 14.0.0, fields FISHCOLLECTIVESPACESCFG, FISHPREVIEWRESOLUTION(value), FISHPREVIEWRESOLUTION(element) were added
-				# 14.0.1, field FISHANNOTATIONSTATECONFG was added
+			$ishVersion = Get-IshVersion
+			if ($ishVersion.MajorVersion -le 13) { $ishFields.Length | Should Be 31 }
+			if ($ishVersion.MajorVersion -eq 14) { 
+				if($ishVersion.RevisionVersion -eq 0) { $ishFields.Length | Should Be 34 } # 14.0.0, fields FISHCOLLECTIVESPACESCFG, FISHPREVIEWRESOLUTION(value), FISHPREVIEWRESOLUTION(element) were added
+				if($ishVersion.RevisionVersion -eq 1) { $ishFields.Length | Should Be 35 } # 14.0.1, field FISHANNOTATIONSTATECONFG was added		
```

Comment:
Sorry... can you change `-eq` to `-ge` so >= instead of = ... otherwise this test is skipped on upcoming 14.0.2

Reply:
Yes, you are right! Added fix